packages/nvidia_nat_memmachine/tests/test_memory.py

```
@@ -0,0 +1,196 @@
+    config: MemMachineMemoryClientConfig,
+    mock_builder: Mock
+):
+    """Test that ImportError is raised when memmachine package is not installed."""
+    # Mock the import to raise ImportError
+    # We need to patch the import inside the function, so patch where it's imported from
+    import builtins
+    original_import = builtins.__import__
+    
+    def import_side_effect(name, *args, **kwargs):
+        if name == "memmachine":
+            raise ImportError("No module named 'memmachine'")
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)